packages/rancher-alerting/rancher-prom2teams/package.yaml

```diff
@@ -3,4 +3,3 @@ subdirectory: helm
 commit: a1691cce91855536be93311fe6b2a4b95b9ce97c # the commit points to the tag 3.2.3
 version: 101.0.0
 doNotRelease: true
-doNotRelease: true
```

Comment:
good catch, I didn't realize I added this key twice. Only one `doNotRelease` should exist (since the chart will already be released under [`charts/rancher-alerting-drivers/VERSION/charts/prom2teams`](https://github.com/rancher/charts/tree/dev-v2.7/charts/rancher-alerting-drivers/100.0.3/charts/prom2teams))